misc/testerina/modules/testerina-runtime/src/main/java/org/ballerinalang/test/runtime/entity/CoverageReport.java

```diff
@@ -355,18 +380,56 @@ private Document getDocumentFromModule(Module moduleInstance, String sourceFileN
     private Collection<ISourceFileCoverage> modifySourceFiles(Collection<ISourceFileCoverage> sourcefiles) {
         Collection<ISourceFileCoverage> modifiedSourceFiles = new ArrayList<>();
         for (ISourceFileCoverage sourcefile : sourcefiles) {
+            ISourceFileCoverage modifiedSourceFile;
+            List<ILine> modifiedLines;
             if (sourcefile.getName().endsWith(BLANG_SRC_FILE_SUFFIX)) {
-                List<ILine> modifiedLines = modifyLines(sourcefile);
-                ISourceFileCoverage modifiedSourceFile = new PartialCoverageModifiedSourceFile(sourcefile,
-                        modifiedLines);
+                modifiedLines = modifyLines(sourcefile);
+                //Normalize source file package name
+                modifiedSourceFile = new PartialCoverageModifiedSourceFile(sourcefile,
+                        modifiedLines, normalizeFileName(sourcefile.getPackageName()));
                 modifiedSourceFiles.add(modifiedSourceFile);
             } else {
                 modifiedSourceFiles.add(sourcefile);
             }
+
         }
         return modifiedSourceFiles;
     }
 
+    private String normalizeFileName(String fileName) {
+        String orgName = IdentifierUtils.encodeNonFunctionIdentifier(
+                this.module.packageInstance().packageOrg().toString());
+        //Get package instance and traverse through all the modules
+        for (Module module : this.module.packageInstance().modules()) {
+            String sourceRoot;
+            String packageName;
+            if (module.isDefaultModule()) {
+                packageName = IdentifierUtils.encodeNonFunctionIdentifier(
+                        module.packageInstance().packageName().toString());
```

Comment:
We can use `module.moduleName()` which will work for both default and non-default module.